lib/worker.js

```diff
@@ -89,10 +91,13 @@ function validateFileName( request ) {
 		suffix += '_' + request.scaleFactor + 'x';
 	}
 
-	let s_filename = _cypher.createHash( 'md5' ).update( request.url ).digest( 'hex' ) + suffix + '.jpg';
+	const extension = request.format === DEFAULT_FORMAT ? '.jpg' : '.png'
```

Comment:
What happens if `DEFAULT_FORMAT` is updated to `png` at some point in the future?